lib/ioc-volumes

```diff
@@ -0,0 +1,196 @@
+#!/bin/sh
+
+__print_exit () {
+    echo "$1"
+    __stop_jail $fulluuid no_unmount
+    exit 1
+}
+
+# format of $volumes: /path/to/some/host/dir[:/location/in/jail/root[:mount,options]]
+# multiple volumes can be space separated
+__mount_volumes () {
+    local _uuid=$1
+    local _mountpoint _source _dest _full_dest _mount_options
+    local _mounts _mounted _retval
+    local volumes="$(__get_jail_prop volumes ${_uuid})"
+
+    if [ -z "${volumes}" -o "${volumes}" == "none" ] ; then
```

Comment:
Given it is a multiconditional if, a comment what's happening here would be useful.